lib/beaker-benchmark/helpers.rb

```diff
@@ -2,6 +2,7 @@
 require 'fileutils'
 require 'time'
 
+# UPDATED BY SLV
```

Comment:
Why this comment?  Git blame will reveal the responsibility of these changes.

Reply:
I think I initially added it as a easy way to ensure I was using the right version of the file on my test runner. I removed it.